homeassistant/components/vera/__init__.py

```diff
@@ -73,13 +74,18 @@
     config = config_entry.data
 
     # Copy the configuration.yml options into the options.
+    if CONF_LIGHTS in config or CONF_EXCLUDE in config:
```

Comment:
This will overwrite the entry options with config yaml options on each restart. We might want to compare the config yaml options from last start with the current config yaml options and only overwrite on change, since the user might have customized the options after last start.